src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft.WinFx.targets

```diff
@@ -106,7 +106,7 @@
   <Target Name="DesignTimeMarkupCompilation">
 
         <!-- Only if we are not actually performing a compile i.e. we are in design mode -->
-        <CallTarget Condition="'$(BuildingProject)' != 'true'"
+        <CallTarget Condition="'$(BuildingProject)' != 'true' Or $(DesignTimeBuild) == 'true'"
```

Comment:
I think we should just remove the "BuildingProject" directive completely - it just adds confusion to this block as to when it is set/not set. This used to be used to detect legacy design-time builds, but this targets file will not be run in legacy.

Reply:
Is `BuildingProject` not set anywhere ?

Reply:
ok, we'll remove it in a separate pass (https://github.com/dotnet/wpf/issues/1914).